tests/apps/courses/test_open_graph_course.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
[The way we name our test files](https://handbook.openfun.fr/python#test-files), this file could be called something like:
`test_templates_course_detail_opengraph.py`

Reply:
done